### ISSUE TITLE
PrefixLists ignores IPv6 config

### ISSUE THREAD

**Issue by bruno-chavez** [awslabs/landing-zone-accelerator-on-aws#198]
**Describe the bug**
When creating [prefix lists](https://awslabs.github.io/landing-zone-accelerator-on-aws/classes/_aws_accelerator_config.PrefixListConfig.html#addressFamily), LZA ignores the `addressFamily` property and tries to use IPv4, failing on the Build stage because of invalid CIDR ranges.

**To Reproduce**
1. Create a prefixList inside `network-config.yaml` with the following configurations
```yaml
prefixLists:
  - name: cloudflare-ipv6
    deploymentTargets:
      organizationalUnits:
        - Root
    addressFamily: IPv6
    maxEntries: 7
    entries:
      - 2400:cb00::/32
      - 2606:4700::/32
      - 2803:f800::/32
      - 2405:b500::/32
      - 2405:8100::/32
      - 2a06:98c0::/29
      - 2c0f:f248::/32
```
2. Run `AWSAccelerator-Pipeline`
3. This will result in the following error

```
2023-07-05 17:59:12.731 \| warn \| config-validator \| Config file validation failed !!!
--
844 | 2023-07-05 17:59:12.748 \| warn \| config-validator \| network-config.yaml has 7 issues:
845 | [Prefix list cloudflare-ipv6]: entry "2400:cb00::/32" is invalid. Value must be a valid IPv4 CIDR range
846 | [Prefix list cloudflare-ipv6]: entry "2606:4700::/32" is invalid. Value must be a valid IPv4 CIDR range
847 | [Prefix list cloudflare-ipv6]: entry "2803:f800::/32" is invalid. Value must be a valid IPv4 CIDR range
848 | [Prefix list cloudflare-ipv6]: entry "2405:b500::/32" is invalid. Value must be a valid IPv4 CIDR range
849 | [Prefix list cloudflare-ipv6]: entry "2405:8100::/32" is invalid. Value must be a valid IPv4 CIDR range
850 | [Prefix list cloudflare-ipv6]: entry "2a06:98c0::/29" is invalid. Value must be a valid IPv4 CIDR range
851 | [Prefix list cloudflare-ipv6]: entry "2c0f:f248::/32" is invalid. Value must be a valid IPv4 CIDR range
852 | error Command failed with exit code 1.
```

**Expected behavior**
It was expected to not fail and create a IPv6 prefix list instead.

**Please complete the following information about the solution:**

- [X] Version: (SO0199-networkprep) Landing Zone Accelerator on AWS. Version 1.4.2.
- [X] Region: us-west-2
- [ ] Was the solution modified from the version published on this repository?
- [ ] If the answer to the previous question was yes, are the changes available on GitHub?
- [x] Have you checked your [service quotas](https://docs.aws.amazon.com/general/latest/gr/aws_service_limits.html) for the sevices this solution uses?
- [ ] Were there any errors in the CloudWatch Logs?

**Screenshots**
If applicable, add screenshots to help explain your problem (please **DO NOT include sensitive information**).

**Additional context**

**Comment by bhkhatri221** [comment]
Hello @bruno-chavez, thank you for identifying this! I have created an issue for our team to investigate this behavior, I will update this issue when I have more information.

Thank you again for your interest and support of the solution!

**Comment by awsclemj** [comment]
Hello,

A fix for this issue has been included in our [v1.5.0 release](https://github.com/awslabs/landing-zone-accelerator-on-aws/releases/tag/v1.5.0). I will be closing this issue as resolved. Thanks for your support of Landing Zone Accelerator!
